MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
> Should SDL2.dll be included in the DesktopGL template?

Well... at the very least when you build the SDL2.dll should appear in your output bin folder.  It shouldn't be up to the user to hunt down the SDL2.dll for themselves.

We could add some custom scripting into our current `MonoGame.Content.Builder.targets` that detects if we're building for 32bit or 64bit and ensures the right SDL2 assembly is added to the build output.

This make sense?  @dellis1972 @KonajuGames ?